app/src/main/AndroidManifest.xml

```diff
@@ -49,7 +49,6 @@
             android:name="net.bible.android.view.activity.page.MainBibleActivity"
             android:configChanges="keyboardHidden|screenSize|locale|uiMode"
             android:windowSoftInputMode="stateHidden|adjustPan"
-            android:launchMode="singleTask"
```

Comment:
Hmm this has opportunity to break many things. We in many ways rely on the fact that there's only 1 instance of MainBibleActivity.